src/util/ssa_expr.h

```diff
@@ -139,6 +139,21 @@ class ssa_exprt:public symbol_exprt
   /* Used to determine whether or not an identifier can be built
    * before trying and getting an exception */
   static bool can_build_identifier(const exprt &src);
+
+  void check(const validation_modet vm = validation_modet::INVARIANT) const
+  {
+    DATA_CHECK(!has_operands(), "SSA expression do not have operands");
```

Comment:
Nit: According to the coding standard (`cbmc/CODING_STANDARD.md`), if possible the messages in invariants should be "should" statements that describe what should hold, e.g. "SSA expression should have operands".